Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;net47;netstandard1.3;netstandard1.6;netstandard2.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net47;netstandard1.3;netstandard1.6;netstandard2.0;netstandard2.1;netcoreapp2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
If we target just `.NET Standard 2.1`, isn't that enough?

Reply:
According to [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=netcore-1.0) it's available from netcore1.0 and netstandard2.1

Reply:
Which means we don't need to explicitly change the targets.

Reply:
As dotnet core 3.0 is the first platform to implement netstandard2.1 and unit test projects must target a platform, i.e. they cannot target netstandard, I agree it should not be necessary to change the targets.

Reply:
There's a dicrepancy about the availability of ValueTask and ValueTask<T> in .netstandard and .netcore. They are not equally available (or unavailable). Thus the two new targets.

Reply:
Ahh, I just assumed that `ValueTask` `ValueTask<T>` was introduced at the same time, but [`ValueTask`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask?view=netcore-2.1) is not available until netcore2.1

What's the difference in availability between netstandard2.1 and netcore2.1?

Reply:
As for .netstandard, both are just available from 2.1. As for core, ValueTask<T> has always been around (according to docs) while ValueTask is around since .net core 2.1.

However, it is always possible to reference the discrete library for all unsupported platforms. You'd still have to import it based on target framework, but you could get rid of the #if directives in the code. I didn't do that because taking additional dependencies as part of a pull request is almost always and for every maintainer a no-go.

Reply:
So what if just stick to .NET Standard 2.1?